notebooks/06_Serving/06_02_kfserving_sdk.ipynb

```diff
@@ -0,0 +1,256 @@
+{
+ "cells": [
```

Comment:
Can you add a section to use S3 as well? Lots of user ask how to load model from S3.

Reply:
If you can not easily figure this out, let me know. I can find previous example for you.

Reply:
Yeah we could, that will be proposed in the next PR.